fluent-plugin-pan-anonymizer.gemspec

```diff
@@ -4,7 +4,7 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 
 Gem::Specification.new do |spec|
   spec.name          = "fluent-plugin-pan-anonymizer"
-  spec.version       = "0.0.1"
+  spec.version       = "0.0.1.1"
```

Comment:
ditto

Reply:
Incremented the version number due to changes.